### PR TITLE
leaderboard: describe API routes to get historical data

### LINKED ISSUE #69: Add links to API endpoints that can be used to get test results per orch for leaderboard
Users should be able to view historical test results per orchestrator, but that is not available in the leaderboard frontend yet. Until that is available in the frontend, we should at least describe how someone can view those test results by using the API endpoint directly in the leaderboard FAQ.

### PR DESCRIPTION
**Pull request by kyriediculous**
Fixes #69

### CODE REVIEW

File: docs/source/reference/leaderboard_faq.rst [hunk 1]
```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
...
 - Review the :doc:`bandwidth_requirements` and consider upgrading your bandwidth 
+
+How can I get historical test results for an orchestrator using the Leaderboard API ? 
+*************************************************************************************
+
+- To get aggregated stats for an orchestrator since a particular timestamp use:
+
+::
+
+    curl https://leaderboard-serverless-git-nv-api-rework.livepeerorg.vercel.app/api/aggregated_stats?orchestrator=<ORCHESTRATOR>?since=<TIMESTAMP>?region=<REGION>
```
Comment (yondonfu) [inline docs/source/reference/leaderboard_faq.rst:157]: Prod URL instead of preview URL for API?

File: docs/source/reference/leaderboard_faq.rst [hunk 2]
```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
...
+
+- <ORCHESTRATOR> the orchestrator's Ethereum address to get the metrics for.
+- <TIMESTAMP> the timestamp at which to evaluate the query. This defaults to the last 24 hours, to get the aggregated stats for all time you can use `since=1`.
+- <REGION> the region to get results for, returns all regions by default.
+
+- To get each individual test entry for an rochestrator since a particular timestmap use:
```
Comment (yondonfu) [inline docs/source/reference/leaderboard_faq.rst:163]: rorchestrator typo

File: docs/source/reference/leaderboard_faq.rst [hunk 3]
```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
...
+How can I get historical test results for an orchestrator using the Leaderboard API ? 
+*************************************************************************************
+
+- To get aggregated stats for an orchestrator since a particular timestamp use:
+
+::
+
+    curl https://leaderboard-serverless-git-nv-api-rework.livepeerorg.vercel.app/api/aggregated_stats?orchestrator=<ORCHESTRATOR>?since=<TIMESTAMP>?region=<REGION>
+
+- <ORCHESTRATOR> the orchestrator's Ethereum address to get the metrics for.
```
Comment (yondonfu) [inline docs/source/reference/leaderboard_faq.rst:159]: Surround bracketed variables i.e. `<ORCHESTRATOR>` with code formatting?

File: docs/source/reference/leaderboard_faq.rst [hunk 4]
```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
 A few things you can explore to improve the speed of data upload/download:
 
 - Review the :doc:`bandwidth_requirements` and consider upgrading your bandwidth 
+
+How can I get historical test results for an orchestrator using the Leaderboard API ? 
+*************************************************************************************
+
+- To get aggregated stats for an orchestrator since a particular timestamp use:
```
Comment (yondonfu) [inline docs/source/reference/leaderboard_faq.rst:153]: Describe what aggregated means?

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: docs/source/reference/leaderboard_faq.rst (hunk 1 above)
Comment (kyriediculous) [reply docs/source/reference/leaderboard_faq.rst]: fixed in 74b5828d64570b09d599a456930e675c8b3a1026

File: docs/source/reference/leaderboard_faq.rst (hunk 2 above)
Comment (kyriediculous) [reply docs/source/reference/leaderboard_faq.rst]: fixed in 74b5828d64570b09d599a456930e675c8b3a1026

File: docs/source/reference/leaderboard_faq.rst (hunk 3 above)
Comment (kyriediculous) [reply docs/source/reference/leaderboard_faq.rst]: fixed in 74b5828d64570b09d599a456930e675c8b3a1026

File: docs/source/reference/leaderboard_faq.rst (hunk 4 above)
Comment (kyriediculous) [reply docs/source/reference/leaderboard_faq.rst]: fixed in 74b5828d64570b09d599a456930e675c8b3a1026

**Review by yondonfu — APPROVED** [review]
LGTM. Pushed a few formatting + typo fixes.

**Pull request merged**